AUTHORS

```diff
@@ -983,4 +984,23 @@ Elias Basler <e.e.basler@protonmail.com>
 Brandon David <brandon.david@zoho.com>
 Abhay_Dhiman <abhaysdhimans@gmail.com>
 Tasha Kim <jae_young_kim@brown.edu>
-Ayush-Malik <ayushmalik779@gmail.com>
+Ayush Malik <ayushmalik779@gmail.com>
+Devesh Sawant <devesh47cool@gmail.com>
+Wolfgang Stöcher <wolfgang@stoecher.com>
+Sudeep Sidhu <sudeepmanilsidhu@gmail.com>
+foice <foice.news@gmail.com>
+Ben Payne <ben.is.located@gmail.com>
+Muskan Kumar <31043527+muskanvk@users.noreply.github.com>
+noam simcha finkelstein <noam.finkelstein@protonmail.com>
+Garrett Folbe <gmfolbe@yahoo.com>
+Islam Mansour <is3mansour@gmail.com>
+Sayandip Halder <sayandiph4@gmail.com>
+Shubham Agrawal <shubham.ag6845@gmail.com>
+numbermaniac <5206120+numbermaniac@users.noreply.github.com>
```

Comment:
numbermaniac wants to be recorded as numbermaniac: https://github.com/sympy/sympy/pull/20066#issuecomment-703041963